### PR TITLE
[eap-additional-testsuite] : Updating the README.md

### PR DESCRIPTION
**Pull request by panossot**
Adding the info about building eap7 tests and how to use it with git SubModule on the servers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +30,35 @@ Testing EAP or Wildfly with specific JDK version
...
+
+PROCEDURE TO FOLLOW WHEN THE GIT SUBMODULES ARE ACTIVATED ON THE SERVERS
+------------------------------------------------------------------------
+1. Add your test at Eap Additional Testsuite and create a PR.
+2. Add your code development/fix at the server and include your eap-additional-testsuite PR as follows :
+   - cd testsuite (go to the testsuite directory of the server)
+   - git submodule update --init --remote (update your submodule)
+   - cd eap-additional-testsuite (go to the eap-additinal-testsuite directory of the server)
+   - git fetch origin refs/pull/NUM OF YOUR EAP ADDITIONAL TESTSUITE PR/head && git checkout FETCH_HEAD
+   - cd ../..
```
Comment (wolfc) [inline README.md:59]: The hash needs to be committed as well.

File: README.md [hunk 2]
```diff
@@ -22,6 +30,35 @@ Testing EAP or Wildfly with specific JDK version
...
+
+
+PROCEDURE TO FOLLOW WHEN THE GIT SUBMODULES ARE ACTIVATED ON THE SERVERS
+------------------------------------------------------------------------
+1. Add your test at Eap Additional Testsuite and create a PR.
+2. Add your code development/fix at the server and include your eap-additional-testsuite PR as follows :
+   - cd testsuite (go to the testsuite directory of the server)
+   - git submodule update --init --remote (update your submodule)
+   - cd eap-additional-testsuite (go to the eap-additinal-testsuite directory of the server)
+   - git fetch origin refs/pull/NUM OF YOUR EAP ADDITIONAL TESTSUITE PR/head && git checkout FETCH_HEAD
```
Comment (wolfc) [inline README.md]: git fetch origin refs/pull/NUM/head

File: README.md [hunk 3]
```diff
@@ -22,6 +30,35 @@ Testing EAP or Wildfly with specific JDK version
...
+
+FUTURE EXTENSIONS
+-----------------
+Use it with every server as a git submodule.
+
+
+PROCEDURE TO FOLLOW WHEN THE GIT SUBMODULES ARE ACTIVATED ON THE SERVERS
+------------------------------------------------------------------------
+1. Add your test at Eap Additional Testsuite and create a PR.
+2. Add your code development/fix at the server and include your eap-additional-testsuite PR as follows :
```
Comment (wolfc) [inline README.md:53]: Markdown has options for designating pieces as command line syntax.

File: README.md (hunk 1 above)
Comment (panossot) [inline README.md:59]: Will not it be commited with the PR to the server? Should I add an instruction about git add testsuite/eap-additional-testsuite?

File: README.md (hunk 3 above)
Comment (panossot) [inline README.md:53]: Used `script` for the syntax.

**Comment by panossot** [comment]
Closing. Outdated.

**Pull request closed without merging**